src/crypto/tls/conn.go

```diff
@@ -133,6 +133,7 @@ type Conn struct {
 		// The client's state, used in case of HRR.
 		innerRandom []byte         // ClientHelloInner.random
 		publicName  string         // ECHConfig.contents.public_name
+		configId    uint8          // ECHConfig.contents.key_config.config_id
 		suite       echCipherSuite // ClientECH.cipher_suite
```

Comment:
Throughout the project, the type needs a rename:
```suggestion
		suite       hpkeSymmetricCipherSuite // ClientECH.cipher_suite
```